src/isotools/_transcriptome_io.py

```diff
@@ -1126,7 +1137,7 @@ def import_ref_transcripts(fn, transcriptome: Transcriptome, file_format, chromo
         exons, transcripts, gene_infos, cds_start, cds_stop, skipped = _read_gff_file(fn, chromosomes, **kwargs)
 
     if skipped:
-        logger.info('skipped the following categories: %s', skipped)
+        logger.info('skipped the following categories: %s', skipped.keys())
```

Comment:
You haven't changed `skipped` from the gff import function, so this will cause an error